cmp/internal/value/format.go

```diff
@@ -0,0 +1,258 @@
+// Copyright 2017, The Go Authors. All rights reserved.
```

Comment:
Is there some way to convince Git and/or GitHub to show this file as a diff on the file it was extracted from?

It looks like this file should be a pretty small delta, but that's hard to see visually in this code review tool...